app/javascript/http_api/api.js

```diff
@@ -24,11 +24,19 @@ const API = {
 
 export default API;
 
+function apiOnly(url) {
+  if (!url.match(/^\/api/)) {
```

Comment:
`\/api\/` (test the trailing slash, to make sure UI controllers can still *start* with api)

and this also needs to match `http://example.com/api/`, because of the way our rest api returns `href`s.

Reply:
`http://example.com/api` is also a valid API endpoint, so I guess the regex should be a little bit more complicated